### PR TITLE
Added ui for customer and clerk

### PR DESCRIPTION
**Pull request by bustamonica**

### CODE REVIEW

File: src/ca/ubc/cs304/database/AddTablesAndData.sql [hunk 1]
```diff
@@ -35,7 +35,7 @@ CREATE TABLE VehicleType(
 
 CREATE TABLE Vehicle(
     vID INTEGER,
-    vLicense VARCHAR(255) DEFAULT 'XXX-XXX',
+    vLicense VARCHAR(255),
```
Comment (ayapcck) [inline src/ca/ubc/cs304/database/AddTablesAndData.sql:38]: Why was this removed

**Review by ayapcck — APPROVED** [review]

**Review by taran1598 — APPROVED** [review]
